sormas-ui/src/main/java/de/symeda/sormas/ui/samples/AdditionalTestController.java

```diff
@@ -57,9 +57,8 @@ public void onCommit() {
 
 	public void openEditComponent(AdditionalTestDto dto, Runnable callback) {
 
-		AdditionalTestDto newDto = FacadeProvider.getAdditionalTestFacade().getByUuid(dto.getUuid());
 		AdditionalTestForm form = new AdditionalTestForm(FacadeProvider.getSampleFacade().getSampleByUuid(dto.getSample().getUuid()), false);
-		form.setValue(newDto);
+		form.setValue(dto);
```

Comment:
Why was this switched to dto instead of newDto?
Getting the latest version of the dto is what we are always doing when starting to edit an entry, so if there is a reason to do it differently here, we need to at least have a comment. In this case it would also not be necessary to do the getByUuid call, done above.

Reply:
we were doing some testing if it affects the detached entity issues and forgot to revert it
will revert it now